[2022-09-04T16:38:03.557+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: first_airflow_dag.save_datetime scheduled__2022-09-04T16:37:00+00:00 [queued]>
[2022-09-04T16:38:03.563+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: first_airflow_dag.save_datetime scheduled__2022-09-04T16:37:00+00:00 [queued]>
[2022-09-04T16:38:03.564+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-04T16:38:03.565+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-09-04T16:38:03.566+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-04T16:38:03.574+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): save_datetime> on 2022-09-04 16:37:00+00:00
[2022-09-04T16:38:03.579+0000] {standard_task_runner.py:52} INFO - Started process 4139 to run task
[2022-09-04T16:38:03.581+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_***_dag', 'save_datetime', 'scheduled__2022-09-04T16:37:00+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp1ttoqgdv', '--error-file', '/tmp/tmp1mafly3w']
[2022-09-04T16:38:03.584+0000] {standard_task_runner.py:80} INFO - Job 108: Subtask save_datetime
[2022-09-04T16:38:03.629+0000] {task_command.py:371} INFO - Running <TaskInstance: first_airflow_dag.save_datetime scheduled__2022-09-04T16:37:00+00:00 [running]> on host c56acb3a0493
[2022-09-04T16:38:03.671+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_***_dag
AIRFLOW_CTX_TASK_ID=save_datetime
AIRFLOW_CTX_EXECUTION_DATE=2022-09-04T16:37:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-04T16:37:00+00:00
[2022-09-04T16:38:03.678+0000] {logging_mixin.py:115} INFO - [{'year': 2022, 'month': 'Sep', 'day': 4, 'time': '16:38:01', 'day_of_week': 'Sun'}]
[2022-09-04T16:38:03.690+0000] {logging_mixin.py:115} INFO -    year month  day      time day_of_week
0  2022   Sep    4  16:38:01         Sun
[2022-09-04T16:38:03.692+0000] {logging_mixin.py:115} INFO - False
[2022-09-04T16:38:03.695+0000] {taskinstance.py:1902} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/first_dag.py", line 44, in save_datetime
    df.to_csv(csv_path, index=False, mode=df_mode, header=df_header)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/generic.py", line 3482, in to_csv
    storage_options=storage_options,
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/formats/format.py", line 1105, in to_csv
    csv_formatter.save()
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/formats/csvs.py", line 243, in save
    storage_options=self.storage_options,
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/io/common.py", line 707, in get_handle
    newline="",
FileNotFoundError: [Errno 2] No such file or directory: '/Users/hugomufraggi/dev/seed-airflow-test/datetimes.csv'
[2022-09-04T16:38:03.705+0000] {taskinstance.py:1412} INFO - Marking task as FAILED. dag_id=first_***_dag, task_id=save_datetime, execution_date=20220904T163700, start_date=20220904T163803, end_date=20220904T163803
[2022-09-04T16:38:03.711+0000] {standard_task_runner.py:97} ERROR - Failed to execute job 108 for task save_datetime ([Errno 2] No such file or directory: '/Users/hugomufraggi/dev/seed-airflow-test/datetimes.csv'; 4139)
[2022-09-04T16:38:03.755+0000] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-09-04T16:38:03.777+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
